packages/caldar-types/src/enums/customer-type.ts

```diff
@@ -0,0 +1,4 @@
+export enum CustomerType {
+  CONSTRUCTION = 1,
+  PRIVATE,
```

Comment:
When declare other `enums` (i.e. `CustomerRoutes`) we use the "Pascal" casing convention, what do you think about changing this one for it to respect that convention?
```suggestion
  Construction = 1,
  Private,
```